backend/src/models/task.js

```diff
@@ -64,9 +83,9 @@ export class Task {
       "CALL save_subtask(?, ?, ?, ?, ?)",
       [
         subtaskData.subtask_id,
-        subtaskData.task_title,
-        subtaskData.task_description,
-        subtaskData.task_state,
+        subtaskData.subtask_title,
+        subtaskData.subtask_description,
+        subtaskData.subtask_state,
```

Comment:
same for this this I think we should have a separate one for subtasks

Reply:
In taskController.js the function for adding subtasks without a specific taskID still exists and I created a distinct one for specifying the task id, but they both use this same query to the database so I'm wondering what would be the difference between the two?

Reply:
So because we have a table for tasks with task_title, task_description and a table for subtasks with subtask_title, subtask_description, we need a separate function and route for those

Reply:
oh wait sorry my bad, i didn't see the git had the thing collapsed and it looked like this change was for `updateDescription`

Upon expanding, it looks good, sorry bout the confusion